sdk/healthinsights/health-insights-radiologyinsights-rest/README.md

```diff
@@ -48,7 +48,7 @@ Managed Identities can also be used to authenticate through DefaultAzureCredenti
 
 ```ts snippet:ReadmeSampleCreateClient_Node
 import { DefaultAzureCredential } from "@azure/identity";
-import RadiologyInsightsRestClient from "@azure-rest/health-insights-radiologyinsights";
+import RadiologyInsightsRestClient from "@azure-rest/health-insights-radiologyinsights-rest";
```

Comment:
I think the package name needs to be `@azure-rest/health-insights-radiologyinsights` without the `-rest` postfix, unless I missed a reason to rename it?

See other azure-rest packages: https://www.npmjs.com/search?q=%40azure-rest%2F